index.js

```diff
@@ -21,6 +23,30 @@ const transformBabel = src => {
   return result;
 };
 
+
+const transformTs = (src, path) => {
+  const {config} = tsconfig.loadSync(process.cwd());
```

Comment:
Yeah, I think that should work, and would get rid of the tsconfig dependency. Will fix that.